src/index.css

```diff
@@ -1,4 +1,4 @@
-* {
+*:not(.noDefaultTransition) {
     box-sizing: border-box;
```

Comment:
I think `box-sizing: border-box` should be applied to the `*` selector. So move this to a new CSS selector block.

Reply:
Fair enough, didn't cause any issues in my end though.